### ISSUE TITLE
Range of NumberProperty should be used when possible

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/greenhouse-effect#317]
Noted while working on https://github.com/phetsims/greenhouse-effect/issues/311

When you have a NumberProperty in the model that has a range, the view should get that range from the NumberProperty - not from some constant that was used to instantiate the NumberProperty.  The former is more robust, the latter is brittle.

In LayersModelModel.ts (model):

```ts
    this.layersInfraredAbsorbanceProperty = new NumberProperty( INITIAL_ABSORPTION_PROPORTION, {
      range: IR_ABSORBANCE_RANGE,
```

In InfraredAbsorbanceControl.ts (view):

```ts
    // convenience variable
    const infraredAbsorbanceRange = LayerModelModel.IR_ABSORBANCE_RANGE;
```

Yes, the above will work.  But this is preferred in InfraredAbsorbanceControl.ts:

```diff
    // convenience variable
-   const infraredAbsorbanceRange = LayerModelModel.IR_ABSORBANCE_RANGE;
+   const infraredAbsorbanceRange = infraredAbsorbanceProperty.range;
```

**Comment by pixelzoom** [comment]
There's a similar  with SolarIntensityControl.ts, it should be:

```diff
    // convenience variable
-   const solarIntensityProportionRange = SunEnergySource.OUTPUT_PROPORTION_RANGE;
+   const solarIntensityProportionRange = surfaceAlbedoProperty.range;
```

**Comment by pixelzoom** [comment]
Here's another example in InfraredAbsorbanceSoundPlayer.ts:

```tw
  public constructor( irAbsorbanceProperty: NumberProperty, irAbsorbanceRange: Range ) {
```

The second parameter is unnecessary, using `irAbsorbanceProperty.range` should be sufficient.

**Comment by pixelzoom** [comment]
Other examples:

```ts
// SolarIntensitySoundPlayer.ts
  public constructor( solarIntensityProperty: NumberProperty, solarIntensityRange: Range ) {

// SurfaceAlbedoSoundPlayer.ts
  public constructor( surfaceAlbedoProperty: NumberProperty, surfaceAlbedoRange: Range ) {
```

**Comment by jbphet** [comment]
Good suggestion, thanks.  I had kind of forgotten about being able to track range information in the ranged Property types, and this is definitely a nice way to keep things simpler.  I've made the suggested changes in most of the suggested places and (I think) a couple of others I noticed along the way.

There are two places where I did _not_ change it.  One was `ConcentrationSliderSoundGenerator`, since the value passed into this is a derived Property, so it doesn't have range information encapsulated with it.  The other is `TemperatureSoundGeneratorFiltered`, since the range of temperature supported in the model is a superset of those expected in some of the instances, so a smaller range is provided to the sound generator.

@pixelzoom - Does it seem reasonable to leave to two mentioned above as is, or is there some better approach that you're aware of?  If not, feel free to close.
